service/insights_v3.go

```diff
@@ -0,0 +1,24 @@
+package service
+
+import (
+	. "github.com/talbright/go-desk/types"
+	. "github.com/talbright/go-desk/resource"
+	"net/http"
+)
+
+type InsightsV3Service struct {
+	client *Client
+}
+
+func (s *InsightsV3Service) Report(query *InsightsV3Query) (*InsightsV3Report, *http.Response, error) {
```

Comment:
Even if not utilized by the desk API, the client would still follow the `X-Rate-Limit-Reset` after receiving a 429 even if it has no effect on concurrency limit